test/org/sagebionetworks/bridge/stormpath/StormpathAccountDaoTest.java

```diff
@@ -80,7 +83,47 @@ public void getAllAccounts() {
     }
     
     @Test
-    public void returnsNulWhenThereIsNoAccount() {
+    public void getStudyPagedAccounts() {
+        List<String> newAccounts = Lists.newArrayList();
+        try {
+            PagedResourceList<AccountSummary> accounts = accountDao.getPagedAccountSummaries(study, 0, 5);
+            // This test requires 6 accounts be present (one more than a page so we can verify the results are capped)
+            // API directories already have 3-6 accounts. They don't need to be verified, consented, etc.
+            if (accounts.getTotal() < 6) {
+                for (int i=0; i < (6-accounts.getTotal()); i++) {
+                    String random = RandomStringUtils.randomAlphabetic(5);
+                    String email = "bridge-testing+"+random+"@sagebridge.org";
+                    SignUp signUp = new SignUp(email, PASSWORD, Sets.newHashSet(TEST_USERS), null);
+                    accountDao.signUp(study, signUp, false);
+                    newAccounts.add(email);
+                }
+                accounts = accountDao.getPagedAccountSummaries(study, 0, 5);
+            }
```

Comment:
Yes, that's not right. Changed to verify at least six first, and then retrieve 5 accounts only.